theseus_gui/src/pages/Settings.vue

```diff
@@ -1,7 +1,471 @@
+          class="input installation-input"
+          placeholder="-Xms10G -Xmx10G -XX:+UseG1GC -XX:+ParallelRefProcEnabled"
+        />
+      </div>
+      <div class="settings-group">
+        <h3>Environment Arguments</h3>
+        <input
+          v-model="settings.custom_env_args"
+          type="text"
+          class="input installation-input"
+          placeholder="ENV1=TEST ENV2=TEST2"
+        />
+      </div>
+      <hr class="card-divider" />
+      <div class="settings-group">
+        <div class="sliders">
+          <span class="slider">
+            Minimum Memory
+            <Slider v-model="settings.memory.minimum" :min="1000" :max="8200" :step="10" />
+          </span>
+          <span class="slider">
+            Maximum Memory
+            <Slider v-model="settings.memory.maximum" :min="1000" :max="8200" :step="10" />
+          </span>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Window Size</h2>
+      <div class="settings-group">
+        <div class="settings-group">
+          <div class="sliders">
+            <span class="slider">
+              Width
+              <Slider v-model="settings.game_resolution[0]" :min="400" :max="2562" :step="2" />
+            </span>
+            <span class="slider">
+              Height
+              <Slider v-model="settings.game_resolution[1]" :min="400" :max="2562" :step="2" />
+            </span>
+          </div>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Launcher Settings</h2>
+      <div class="settings-group">
+        <h3>Console</h3>
+        <div class="toggle-setting">
+          Maximum Concurrent Downloads
+          <input
+            v-model="settings.max_concurrent_downloads"
+            type="number"
+            name="concurrent-downloads"
+            class="concurrent-downloads"
+          />
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Commands</h2>
+      <div class="settings-group">
+        <div class="toggle-setting">
+          Pre Launch
+          <input ref="javaArgs" v-model="settings.hooks.pre_launch" type="text" class="input" />
+        </div>
+        <div class="toggle-setting">
+          Wrapper
+          <input ref="javaArgs" v-model="settings.hooks.wrapper" type="text" class="input" />
+        </div>
+        <div class="toggle-setting">
+          Post Launch
+          <input ref="javaArgs" v-model="settings.hooks.post_exit" type="text" class="input" />
+        </div>
+      </div>
+    </Card>
+    <Button color="primary" class="save-btn" @click="saveSettings"><SaveIcon />Save changes</Button>
   </div>
 </template>
+
+<style lang="scss">
+.active {
+  background: var(--color-brand-highlight) !important;
+}
+
+.concurrent-downloads {
+  min-height: 40px;
+  max-width: 4.2rem !important;
+  text-align: center;
+}
```

Comment:
this is not styled at all for me 
![image](https://user-images.githubusercontent.com/18202329/232964903-db2bdd83-fc29-480a-9ce1-ef0cc1298682.png)

This should also be a slider (sorry for not mentioning earlier) with a minimum of 1 and a maximum of 100

Reply:
For the styling to come through, you need to install the updated Omorphia package. Your local node_modules probably still has 4.4.